### PR TITLE
init project

### PR DESCRIPTION
**Pull request by thanhgiong195**

### CODE REVIEW

**Comment by thanhgiong195** [comment]
ready

File: .framgia-ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    command: brakeman -o .framgia-ci-reports/brakeman.html -o .framgia-ci-reports/brakeman.json
+  reek:
+    command: reek --format html > .framgia-ci-reports/reek.html
+  rubocop:
+    command: rubocop --require rubocop/formatter/checkstyle_formatter --format RuboCop::Formatter::CheckstyleFormatter --no-color --rails --out .framgia-ci-reports/rubocop.xml app/ lib/
+  scss-lint:
+    command: scss-lint --require=scss_lint_reporter_checkstyle --format=Checkstyle --out=.framgia-ci-reports/scss-lint.xml app/assets/stylesheets/
+  eslint:
+    command: eslint --format=checkstyle --output-file=.framgia-ci-reports/eslint.xml app/
+    ignore: true
```
Comment (dieunb) [inline .framgia-ci.yml]: xóa đi

**Comment by thanhgiong195** [comment]
ready

**Comment by thanhgiong195** [comment]
ready

**Comment by dieunb** [comment]
ready

**Comment by thanhgiong195** [comment]
ready

**Comment by dieunb** [comment]
commented

**Comment by thanhgiong195** [comment]
ready

**Pull request merged**